cmd/juju/crossmodel/remove.go

```diff
@@ -51,6 +62,13 @@ func (c *removeCommand) Info() *cmd.Info {
 	}
 }
 
+// SetFlags implements Command.SetFlags.
+func (c *removeCommand) SetFlags(f *gnuflag.FlagSet) {
+	f.BoolVar(&c.force, "force", false, "remove the offer as well as any relations to the offer")
```

Comment:
I think you'd need to call SetFlags on the higher command in hierachy, i.e. ControllerCommandBase...

In fact, looking at this code, have you tested with -c (i.e. running on a different controller)?

Reply:
Add call to base struct
We only really test login cmd extensively with -c; added a basic test here

Reply:
Sure, we do not have many tests and tha's why we forget to call super- methods. Which leads to unexpected bugs.

We should try to make sure that at least one unit tests has a -c on the commands that support it.